msal/src/main/java/com/microsoft/identity/client/statemachine/Error.kt

```diff
@@ -82,7 +82,7 @@ class PasswordIncorrectError(
     override val errorMessage: String,
     override val correlationId: String,
     override val errorCodes: List<Int>
-) : GeneralError(errorMessage = errorMessage, error = error, correlationId = correlationId, errorCodes = errorCodes)
+) : Error(errorMessage = errorMessage, error = error, correlationId = correlationId, errorCodes = errorCodes)
```

Comment:
Why are there so many identical classes?

Reply:
You can easily write one Error class and then use different Strings or enums to represent different error codes within that Error object